### DISCUSSION TITLE
It's a no for me.

### DISCUSSION THREAD

**Discussion by Defplayr** [rcambrj/sovol-dryer-firmware#1] [Reports of success / failure]
Sovol SV01
HC32F005x6xx
BAITE BTE13-007 adapter - seen as a [CP2102 bridge](https://www.taydaelectronics.com/datasheets/files/A-1991.pdf)

Programmer would start, and on pressing 'Upload', would crash after three lines of text. No apparent communication with MCU. SV01 functioning unchanged. Running on Win10 22H2 bog standard PC.

Tried: 
Connecting adapter RTS to MCU NRST
Connecting adapter DTR to MCU NRST
Connecting adapter Ground to MCU NRST
FIFO on/off in adapter driver
Different baud rates
Running programmer in Admin and compatability mode.
All had no effect.

![20241204_184815](https://github.com/user-attachments/assets/65adc33a-8165-4576-bcd0-9b464e5898a9)
![20241204_185006](https://github.com/user-attachments/assets/e35f0539-e154-421a-ba9f-8ea6bb6a35fb)

**Comment by rcambrj** [comment]
Thanks for the report. You're the first! 🥳 Give yourself a pat on the back.

Your experience sounds exactly like what would happen with the Adafruit FTDI Friend. I'll add your adapter to the _not working_ list.

I recommend getting the [DSD TECH SH-U09C5 FT232NL](https://www.amazon.nl/-/en/dp/B07WX2DSVB). Let me know how it goes 🙂

Edit: wait, you wrote that the programmer would _crash_? That's new.

**Reply by Defplayr** [reply]
Thanks for the reply! My one works fine with 8266, Arduino (& clones) and putting Tasmota on just about everything, but never turn down an upgrade! I'll let you know.

**Reply by rcambrj** [reply]
Same! I've been using the FTDI Friend since forever... I think it's the fussy chinese software 🙂

**Reply by Defplayr** [reply]
Success! Changed programmer to a: [DSD TECH SH-U09C5 USB to TTL UART Converter Cable with FTDI Chip Support](https://www.amazon.co.uk/dp/B07WX2DSVB?ref=ppx_yo2ov_dt_b_fed_asin_title). Thank you. There's some interest in this mod on the Reddit r/Bambu & r/3Dprinting threads. Do you mind if I link you on a post, and do you have an affiliate link for the SH-U09C5? (might be worth getting one!)

**Reply by rcambrj** [reply]
YAY! It's weird and quirky, but whatever.

Please go ahead and post 🙂
